### ISSUE TITLE
What are your experimental techniques?

### ISSUE THREAD

**Issue by xiaosenho** [arcaninez/ODM3D#3]
CMKD mentioned in the paper that they used 18k additional data for 10 epochs of deep pre-training on the test set, and used 42k full original data sets for training to achieve sota performance. Can you achieve performance improvement on the test without mentioning any additional techniques in your paper?

![Image](https://github.com/user-attachments/assets/0ad338fd-8f78-4e68-8a0d-a89bf7afd38a)

**Comment by charmeleonz** [comment]
Hi, as I recall, while the authors of CMKD describe such training techniques in their paper, they did not implement them in their official repository. You should be able to obtain close performance without such techniques. Our experiments follow the implementations of CMKS's repository, so we do not use those techniques.

Also please note that their described pre-training is not done on the test set.

**Comment by arcanienz** [comment]
Hi, as I recall, while the authors of CMKD describe such training techniques in their paper, they did not implement them in their official repository. You should be able to obtain close performance without such techniques. Our experiments follow the implementations of CMKS's repository, so we do not use those techniques.

Also please note that their described pre-training is not done on the test set.
